packages/jest-emotion/test/preact.test.js

```diff
@@ -0,0 +1,43 @@
+import { h } from 'preact'
+import render from 'preact-render-to-json'
+import prettyFormat from 'pretty-format'
+import * as emotion from 'emotion'
+import { createSerializer } from '../src'
+const { ReactElement, ReactTestComponent, DOMElement } = prettyFormat.plugins
+
+/** @jsx h */
+
+describe('jest-emotion with preact', () => {
+  const emotionPlugin = createSerializer(emotion)
+
+  const divStyle = emotion.css`
+    color: red;
+  `
+  const svgStyle = emotion.css`
+    width: 100%;
+  `
+
+  it('replaces class names and inserts styles into preact test component snapshots', () => {
+    const tree = render(
+      <div class={divStyle}>
```

Comment:
is `preact` accepting `class` instead of `className`?

There is a linting problem found in this file:
> Unknown property 'class' found, use 'className' instead  react/no-unknown-property

If `class` is correct in `preact` you can just disable this rule for this file (with top file comment or smth)

Reply:
Thanks. Fixed. And yes, `preact` takes all the html style instead of js style (`class` and `for`, for example)